plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,751 @@
+  lockwait:
+    description:
+      - Maximal time to wait for the global lock (minutes).
+    type: int
+  mailnotification:
+    description:
+      - Specify when to send a notification mail
+    choices: ['always', 'failure']
+    type: str
+  mailto:
+    description:
+      - Comma-separated list of email addresses or users that should receive email notifications.
+    type: str
+  maxfiles:
+    description:
+      - Maximal number of backup files per guest system.
+    type: int
+  mode:
+    description:
+      - Backup mode.
+    choices: ['snapshot', 'suspend', 'stop']
+    type: str
+  node:
+    description:
+      - Only run if executed on this node.
+    type: str
+  notes_template:
+    description:
+      - Template string for generating notes for the backup(s).
+      - It can contain variables which will be replaced by their values.
+      - Currently supported are V({{cluster}}), V{({guestname}}), V({{node}}), and V({{vmid}}),but more might be added in the future.
+      - Needs to be a single line, newline and backslash need to be escaped.
+    type: str
+  performance:
+    description:
+      - Other performance-related settings.
+    type: str
+  pigz:
+    description:
+      - Use pigz instead of gzip when V(N>0).
+      - V(N=1) uses half of cores, V(N>1) uses N as thread count.
+    type: int
+  pool:
+    description:
+      - Backup all known guest systems included in the specified pool.
+      - Can not be use with vmid and pool in same job
+    type: str
+  protected:
+    description:
+      - If true, mark backup(s) as protected.
+    type: bool
+  prune_backups:
+    description:
+      - Use these retention options instead of those from the storage configuration.
+    type: str
+  quiet:
+    description:
+      - Be quiet.
+    type: bool
+  remove:
+    description:
+      - Prune older backups according to 'prune-backups'.
+    type: bool
+  repeat_missed:
+    description:
+      - If V(true), the job will be run as soon as possible if it was missed while the scheduler was not running.
+    type: bool
+  schedule:
+    description:
+      - Backup schedule. The format is a subset of `systemd` calendar events.
+    type: str
+  script:
+    description:
+      - Use specified hook script.
+    type: str
+  starttime:
+    description:
+      - Job Start time.
+    type: str
+  stdexcludes:
+    description:
+      - Exclude temporary files and logs.
+    type: bool
+  stop:
+    description:
+      - Stop running backup jobs on this host.
+    type: bool
+  stopwait:
+    description:
+      - Maximal time to wait until a guest system is stopped (minutes).
+    type: int
+  storage:
+    description:
+      - Store resulting file to this storage.
+    type: str
+  tmpdir:
+    description:
+      - Store temporary files to specified directory.
+    type: str
+  vmid:
+    description:
+      - The ID of the guest system you want to backup.
+      - Can not be use with vmid and pool in same job
```

Comment:
Please note that this IS the `vmid` option. The description here is most likely wrong.
```suggestion
      - Can not be use with O(vmid) and O(pool) in same job.
```

Reply:
Fix in 53df0504d36dc7ee70c350ae3467cca413e0a8b0

Reply:
Same here.